scripts/train.py

```diff
@@ -0,0 +1,41 @@
+import os
+import click
+import pickle
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+from pkspace.utils import trainer
+
+
+@click.command()
+@click.option('--PKSpace', 'dataset_mode', flag_value='PKSpace',
+              default=True)
+@click.option('--PKLot', 'dataset_mode', flag_value='PKLot')
+@click.option('--MLP', 'method', flag_value='MLP',
+              default=True, help='Method to be used for prediction')
+@click.option('--dataset_dir', required=True,
+              help='Directory of dataset for model to be trained on')
+@click.option('--output', default=None,
+              help='Name of output file for trained model')
+def train(dataset_mode, method, dataset_dir, output):
```

Comment:
@rstevanak Also every argument/option should have a abbreviation option, something like, `--output, -o`.